charts/consul/test/unit/controller-clusterrole.bats

```diff
@@ -18,6 +18,149 @@ load _helpers
       yq 'length > 0' | tee /dev/stderr)
   [ "${actual}" = "true" ]
 }
+#--------------------------------------------------------------------
+# rules
+
+@test "controller/ClusterRole: sets create, delete, get, list, patch, update and watch access to all CRDs in the consul.hashicorp.com api group" {
```

Comment:
added these three tests that were missing from existing code, but are unrelated to webhook certs.

Reply:
Typically, we don't add tests for YAML that doesn't have any templating logic around it (the case for these rules I think) because we don't see they add that much value that would justify the effort of writing them (writing bats is kind of tedious). I think it's fine to keep these here though if you found they were valuable!